[2025-05-09T15:52:09.441+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.upload_to_hdfs manual__2025-05-09T15:50:51.637660+00:00 [queued]>
[2025-05-09T15:52:09.456+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.upload_to_hdfs manual__2025-05-09T15:50:51.637660+00:00 [queued]>
[2025-05-09T15:52:09.457+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 3
[2025-05-09T15:52:09.477+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): upload_to_hdfs> on 2025-05-09 15:50:51.637660+00:00
[2025-05-09T15:52:09.486+0000] {standard_task_runner.py:57} INFO - Started process 1699 to run task
[2025-05-09T15:52:09.491+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fetch_transaction_data_and_upload_to_hdfs', 'upload_to_hdfs', 'manual__2025-05-09T15:50:51.637660+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/ingestion.py', '--cfg-path', '/tmp/tmppzfpilpz']
[2025-05-09T15:52:09.494+0000] {standard_task_runner.py:85} INFO - Job 32: Subtask upload_to_hdfs
[2025-05-09T15:52:09.563+0000] {task_command.py:415} INFO - Running <TaskInstance: fetch_transaction_data_and_upload_to_hdfs.upload_to_hdfs manual__2025-05-09T15:50:51.637660+00:00 [running]> on host d4afc872a8aa
[2025-05-09T15:52:09.661+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_transaction_data_and_upload_to_hdfs' AIRFLOW_CTX_TASK_ID='upload_to_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-05-09T15:50:51.637660+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-09T15:50:51.637660+00:00'
[2025-05-09T15:52:13.916+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-09T15:52:13.927+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fetch_transaction_data_and_upload_to_hdfs, task_id=upload_to_hdfs, execution_date=20250509T155051, start_date=20250509T155209, end_date=20250509T155213
[2025-05-09T15:52:13.974+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-09T15:52:13.991+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
